src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4032,6 +4032,11 @@ internal void ImportCmdletsFromAssembly(Assembly assembly, PSModuleInfo module)
 {
     if ($psCmdlet.ParameterSetName -eq 'ScriptInputSet')
     {
+        if (!$PSBoundParameters.ContainsKey('cursorColumn'))
```

Comment:
Can you use a default value instead? I think that should work as binding defaults run after binding specified arguments, and it would avoid this test which happens naturally in the parameter binder.

Reply:
I didn't think it would work, but it does.  Definitely simpler.